aten/src/ATen/native/LinearAlgebraUtils.h

```diff
@@ -97,7 +97,7 @@ static inline void batchCheckErrors(std::vector<int64_t>& infos, const char* nam
     } else if (info > 0) {
       if (strstr(name, "svd")) {
         AT_ERROR(name, ": the updating process of SBDSDC did not converge (error: ", info, ")");
-      } else if (strstr(name, "symeig")) {
+      } else if (strstr(name, "symeig") || strstr(name, "syevd")) {
```

Comment:
Let's use the same error message as for `symeig` if LAPACK/MAGMA's algorithm doesn't converge. I couldn't find what kind of input matrix would trigger this though.